storage/interface.go

```diff
@@ -148,6 +176,10 @@ type Appender interface {
 	// faster than adding a sample by providing its full label set.
 	AddFast(ref uint64, t int64, v float64) error
 
+	AddExemplar(l labels.Labels, e exemplar.Exemplar) error
```

Comment:
Some comments would be good here and below, as this is the public interface

Reply:
yes I still need to clean up the interfaces in general, including comments